### ISSUE TITLE
Platform 'auto' not found

### ISSUE THREAD

**Issue by danielholmberg** [moonlight-stream/moonlight-docs#96]
Hi!

I've tried installing, removing an re-installing the package moonlight-embedded and tried multiple different tutorials on how to successfully install Moonlight and stream Steam on my Raspberry Pi 3 Raspbian. 

I'm successfully pairing with my PC but when I try to start a stream I get the following output:
Connect to [MY_PC_IP]...
Platform 'auto' not found

Please, does anyone know how to solve this issue?
Thanks!

**Comment by danielholmberg** [comment]
I now tried to specify the specific platform using the flag -platform with the option 'fake': 
~: moonlight stream -app Steam -platform fake 

and it finally starts Steam on my computer... but it does not stream the image to the Raspberry Pi.  My output now is:

Searching for server...
Connect to [MY_PC_IP]...
Mapping for Sony PLAYSTATION(r)3 Controller (03000....0124) on /dev/input/event2 is incorrect
Initializing platform...done
Resolving hostname...done
Starting RTSP handshake...done
Initializing control stream...done
Initializing video stream...done
Initializing audio stream...done
Initializing input stream...done
Starting control stream...done
Starting video stream...done
Starting audio stream...done
Starting input stream...done
Alsa error from writei: -32

Why doesn't the stream work?

**Comment by XJIOP** [comment]
same on tinkerboard with rockchip

```
# moonlight stream -app Steam
Platform 'auto' not found
```

```
# moonlight stream -app Steam -platform fake
No mapping available for RC for dw_hdmi (1e000000524320666f722064775f6800) on /dev/input/event1
No mapping available for gpio-keys (19000000010000000100000000010000) on /dev/input/event3
No mapping available for rockchip,miniarm-codec Headphones (00000000726f636b636869702c6d6900) on /dev/input/event2
```

```
# cd /dev/input/by-path && ls
platform-ff500000.usb-usb-0:1:1.255-event  
platform-ff980000.hdmi-event  
platform-gpio-keys-event  
platform-sound-simple-card-event
```

how to fix it?
